### ISSUE TITLE
Document `data-resources-url`

### ISSUE THREAD

**Issue by thomasaull** [ionic-team/stencil#2883]
This is a follow up to https://github.com/ionic-team/stencil/issues/903. It seems like the attribute is still not documented (I can't find anything in the official docs at least). I figured it out by checking the generated source. I think this this might be helpful to others, so it should be documented.

Also I would question the practice to close issues which are inactive for a month even if the issue is not resolved yet. I know it's hard to stay on top of all those issues but by auto-closing (resolved or not) important things might get lost…

**Comment by splitinfinities** [comment]
Yep I'm going to denote this as a part of our docs effort. Seems to fit quite well with assets.

**Comment by rwaskiewicz** [comment]
I'm gonna move this to the Stencil Site repo to make sure it gets grouped with similar issues, since we're looking to update the docs around the PR linked in the Issue Summary.
